CHANGELOG.md

```diff
@@ -8,6 +8,18 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [0.3.2] - 2023-01-18
+
+[Full changelog](https://github.com/openfga/openfga/compare/v0.3.1...v0.3.2)
+
+
+### Added
+* OpenTelemetry metrics and tracing `otlp` exporter (#360) - thanks @AlexandreBrg!
```

Comment:
do we want to note this and the optimized ListObjects are released under experimental flag?

Reply:
I thought we already did in a prior release?

It's documented here already
https://openfga.dev/docs/getting-started/setup-openfga#experimental-features